src/store/modules/plex/actions.js

```diff
@@ -68,8 +68,92 @@ export default {
       commit('PLEX_SET_VALUE', ['servers', {}]);
       commit('PLEX_SET_VALUE', ['clients', {}]);
     }
-    const options = PlexAuth.getApiOptions('https://plex.tv/api/resources?includeHttps=1', state.user.authToken, 5000, 'GET');
-    request(options, (error, response, body) => {
+    const clients = PlexAuth.getApiOptions('https://plex.tv/devices.xml?includeHttps=1', state.user.authToken, 5000, 'GET');
+    request(clients, (error, response, body) => {
+      if (!error && response.statusCode === 200) {
+        // Valid response
+        parseXMLString(body, async (err, result) => {
+          if (err) {
+            return reject(err);
+          }
+          var client = 10;
+          for (var index in result.MediaContainer.Device) {
+            const device = result.MediaContainer.Device[index].$;
+            const connections = result.MediaContainer.Device[index].Connection;
+            if (device.provides.indexOf('player') > -1 && device.product != "Plex Web" && connections && connections.length > 0) {
+              console.log(client)
```

Comment:
You might want to add a filter for devices only seen in the last X minutes. Rationale: I only have 3 devices that I've used in the last several days but I have over 20 non-server devices registered in the api and would want to see all of them.
Something like this:
```javascript
let recentWindow = 5 * 60 * 1000; // 5 minutes
let lastSeenDiff = (Math.round((new Date()).getTime() / 1000) - device.lastSeenAt;
if (lastSeenDiff >  recentWindow) {
  continue;
}
```

Reply:
So the reason for this messed up filter is devices.xml returns things liek Plex Synclougne Auth page over and over. But agree on the recently seen. Set it to 20 minutes.